src/population.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+class Population():
+    def __init__(self, solutions, fitness):
+        self.solutions = solutions
+        self.fitness = fitness
+
+    def initial_population(self, population_size, test_size):
+        population = list()
+        for i in range(population_size):
+            test = list()
+            for j in range(test_size):
+                test.append(random.randint(-1000, 1000))
+            population.append(test)
+        return population
```

Comment:
What the hell??

why do you need `self.solutions`?

You have to store all the TestCases in `self.solutions`